CMakeLists.txt

```diff
@@ -49,7 +45,7 @@ find_package(PNG REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(Jasper REQUIRED)
```

Comment:
ZLIB, Japser and PNG are not direct dependencies, Can be removed.

Reply:
done.